docs/recipes/install/common/openchami_node_discovery.tex

```diff
@@ -0,0 +1,47 @@
+% -*- mode: latex; fill-column: 120; -*-
+\OpenCHAMI{} can do dynamic discovery, but here we will do node discovery by YAML file. The YAML file can be imported by the discover command creating a way for adding nodes that do not have redfish capabilities, or when you just want to provide a list of nodes.
+
+% begin_ohpc_run
+% ohpc_comment_header Discovering Nodes \ref{sec:static_node_discovery}
+\begin{lstlisting}[language=bash,keywords={}]
+[sms](*\#*) mkdir -p /opt/workdir/nodes
+[sms](*\#*) export ${compute_prefix^^}_ACCESS_TOKEN=$(bash -lc 'gen_access_token')
+[sms](*\#*) touch /opt/workdir/nodes/node.sh
+[sms](*\#*) cat > /opt/workdir/nodes/node.sh <<'EOF'
```

Comment:
Would it make sense to ship this as a script somewhere else? Either in the openchami RPM or in one of the openhpc RPMS (ohpc-filesystem for example).

We have talked about helper scripts also in the context of warewulf4, but so far it was not necessary.